yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1015,12 +1014,46 @@ func analyzeSchemaInternal(sourceDBConf *srcdb.Source) utils.SchemaReport {
 			checkExtensions(sqlInfoArr, filePath)
 		}
 		checker(sqlInfoArr, filePath)
```

Comment:
Do we need to run`checker` even in cases where the objType is EXTENSION/CONVERSION?

Reply:
Got it yeah makes sense! Maybe there are other issues as well in addition to the CONVERSION